video_creation/final_video.py

```diff
@@ -436,9 +436,11 @@ def on_update_example(progress) -> None:
 
         old_percentage = pbar.n
         pbar.update(100 - old_percentage)
+
     pbar.close()
     save_data(subreddit, filename + ".mp4", title, idx, background_config["video"][2])
+
     print_step("Removing temporary files 🗑")
-    cleanups = cleanup(reddit_id)
-    print_substep(f"Removed {cleanups} temporary files 🗑")
+    cleanup(reddit_id)
     print_step("Done! 🎉 The video is in the results folder 📁")
+    time.sleep(1)  # Prevents the last 2 print_step functions from not showing up
```

Comment:
forget to delete the sleep statement?

Reply:
Actually no, I left that intentionally. As the comment next to sleep says, if it doesn't sleep for a second, the last 2 print_step functions don't show up, like the "Done! 🎉 The video is in the results folder 📁" for example. Haven't really found another way around it, without the sleep, it just exits too quickly.

Reply:
try using flush in the internal print statement (e.g. print..., flush=True)